### DISCUSSION TITLE
Tuya integration DS-03 fan switch

### DISCUSSION THREAD

**Discussion by wilsosw** [home-assistant/frontend#10484] [Automations / scripts / scenes] [answered]
Prior release had a light switch that would not indicate position correctly but did function. HA beta the switch is gone. No light control. Fan switch still exists and indicates and functions correctly as it did prior to beta.

**Comment by frenck** [comment, marked as answer]
This is not a frontend discussion about new frontend feature or frontend design, but a core issue.

Please this issue in the issue tracker here:

<https://github.com/home-assistant/core/issues>

Be sure to check your issue hasn't been reported by someone else already by using the search.

Thanks! 👍 

../Frenck
